job/job.go

```diff
@@ -86,6 +94,30 @@ type Job struct {
 	IsDone bool `json:"is_done"`
 }
 
+type jobType int
+
+const (
+	LocalJob jobType = 0 + iota
+	RemoteJob
+)
+
```

Comment:
@levrado Good idea, but you should be able to just say `iota` rather than `0 + iota`